src/gui/libcloudproviders/libcloudproviders.cpp

```diff
@@ -12,8 +12,8 @@
  * for more details.
  */
 
-#include <cloudproviders/cloudprovidersaccountexporter.h>
-#include <cloudproviders/cloudprovidersproviderexporter.h>
+#include <cloudprovidersaccountexporter.h>
```

Comment:
unrelated

Reply:
It did not want to build locally, so I thought I'd include it...

Reply:
please fix